linera-core/src/worker.rs

```diff
@@ -1120,6 +1131,9 @@ where
             NUM_ROUNDS_IN_CERTIFICATE
                 .with_label_values(&[log_str])
                 .observe(round.0 as f64);
+            if is_confirmed {
+                TRANSACTION_COUNT.with_label_values(&[]).inc();
```

Comment:
Now we want to increase it not just by one, but by the number of transactions in the confirmed block.

I'd just create a variable `confirmed_transactions` and set it to `0` in the other branches. In the `ConfirmedBlock { executed_block }` branch, it would be `block.incoming_messages.len() + block.operations.len()`, where `block = executed_block.block`.

Reply:
Ah ok, sorry. Let me try that